core/src/main/java/hudson/Functions.java

```diff
@@ -304,6 +304,7 @@ public static void initPageVariables(JellyContext context) {
         context.setVariable("resURL",rootURL+getResourcePath());
         context.setVariable("imagesURL",rootURL+getResourcePath()+"/images");
         context.setVariable("divBasedFormLayout", true);
+        context.setVariable("divBasedRunLayout", true);
```

Comment:
used in https://github.com/jenkinsci/workflow-job-plugin/pull/180 to adapt to this change without requiring a core bump